x-pack/plugins/ml/public/application/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -179,6 +185,38 @@ export class JobsList extends Component {
         ),
         render: (item) => <JobIcon message={item} showTooltip={true} />,
       },
+      {
+        field: 'alertingRules',
+        name: (
+          <EuiScreenReaderOnly>
+            <p>
+              <FormattedMessage
+                id="xpack.ml.jobsList.alertingRules.screenReaderDescription"
+                defaultMessage="This column display icons when there are alert rules associated with a job"
```

Comment:
Small typo - should be `displays`

Reply:
Good spot! another column (`auditMessage`) which I've copied from has the same issue, I'll fix both

Reply:
Fixed in https://github.com/elastic/kibana/pull/97626/commits/0747f5f184462de17e7e7cbe40ec6cb68dc88a8c